examples/eng_wasm_contracts/contract_with_eth_calls/Test.json

```diff
@@ -1 +1,20 @@
-[{"constant":false,"inputs":[{"name":"","type":"uint256"},{"name":"","type":"address[]"}],"name":"getBryn","outputs":[],"payable":false,"stateMutability":"nonpayable","type":"function"}]
+[
+	{
+		"constant":false,
+		"inputs":[{"name":"","type":"uint256"},{"name":"","type":"address[]"}],
```

Comment:
(I'm super picky, i know)
please add a bit of spacing after colons `:` and commas `,`
same on line 13

Reply:
fixed in 610284d